src/request-extractors/hapi.ts

```diff
@@ -61,7 +60,7 @@ function attemptToExtractStatusCode(req: hapi.Request) {
 function extractRemoteAddressFromRequest(req: hapi.Request) {
   if (has(req.headers, 'x-forwarded-for')) {
     return req.headers['x-forwarded-for'];
-  } else if (isObject(req.info)) {
+  } else if (is.object(req.info)) {
```

Comment:
It looks like `isObject` is still declared in this file but is no longer needed.

Reply:
It looks like the same is true for `isArray`.